nautobot/ipam/models.py

```diff
@@ -722,21 +791,35 @@ class IPAddress(PrimaryModel, StatusModel):
         "description",
     ]
 
-    objects = IPAddressManager()
+    objects = IPAddressQuerySet.as_manager()
 
     class Meta:
-        ordering = ("address",)  # address may be non-unique
+        ordering = ("host", "prefix_length", "pk")  # address may be non-unique
```

Comment:
I don't think it's desirable to order by `pk`. If we want a consistent ordering for same-address instances, maybe instead add `vrf`, `tenant`, `role` to the ordering instead?